src/sign-in-blazorwasm/BlazorWasm.csproj

```diff
@@ -11,6 +11,8 @@
     <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly" Version="6.0.0" />
     <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly.DevServer" Version="6.0.0" PrivateAssets="all" />
     <PackageReference Include="Microsoft.Authentication.WebAssembly.Msal" Version="6.0.0" />
+    <PackageReference Include="Microsoft.Graph" Version="4.18.0" />

```

Comment:
As talked about on Teams, we want to avoid using the Graph SDK and instead opt for a raw http request to `https://graph.microsoft.com/v1.0/me` so that we can show 1. how MSAL is used to acquire access tokens for a specific audience/scope, and 2. how those access tokens are typically used when calling a protected API.

Graph just happens to be our API, but it's really just a stand-in for `https://contoso.com/api/something/awesome` :)

Reply:
New version looks good, IMO; using the native Blazor functionality to acquire tokens and attach them to downstream API requests.